svc/GridExportService.ts

```diff
@@ -94,14 +94,14 @@ export class GridExportService extends HoistService {
         if (cellCount > withDefault(config.streamingCellThreshold, 100000)) {
             startToast = XH.toast({
                 message: 'Your export is being prepared. Due to its size, formatting will be removed.',
-                icon: Icon.download(),
+                icon: Icon.downloadIcon(),
```

Comment:
this seems confusing to me --is the icon suffix here a mistake?

Reply:
Agreed we should not have a trailing `xxxxIcon` on any of these

Reply:
Not sure how that slipped through... will correct it.